EngineLayer/MetaMorpheusEngine.cs

```diff
@@ -185,11 +248,6 @@ private static void AddComplementaryPeaks(ref double[] experimental_mzs, ref dou
         {
             List<MzPeak> complementaryPeaks = new List<MzPeak>();
 
-            //keep original peeks
```

Comment:
No longer keep original peaks, so one search is conducted with original, and one with only comp ions.

Reply:
Oooh! Clever